### PR TITLE
Add org.tryton.Tryton-74

### PR DESCRIPTION
**Pull request by cedk**
This is the new series following org.tryton.Tryton-72

### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly. 
- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: org.tryton.Tryton-74.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+id: org.tryton.Tryton-74
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '46'
```
Comment (bbhtt) [inline org.tryton.Tryton-74.yml:4]: 47 is latest

File: org.tryton.Tryton-74.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  - python3-tryton.json
+  - name: GoocanvasDependency
+    buildsystem: autotools
+    make-install-args:
+      - pyoverridesdir=/app/lib/python3.11/site-packages/gi/overrides
```
Comment (bbhtt) [inline org.tryton.Tryton-74.yml:17]: 47 has 3.12

File: org.tryton.Tryton-74.yml (hunk 1 above)
Comment (cedk) [reply org.tryton.Tryton-74.yml:4]: goocanvas does not compile on it.

Comment (cedk) [reply org.tryton.Tryton-74.yml:4]: I found a patch to build goocanvas on 47 but now I have this error:
```
Running appstream-compose
bwrap: execvp appstream-compose: No such file or directory
Error: ERROR: appstream-compose failed: Child process exited with code 1
```
while the same manifest works with 46.

Comment (bbhtt) [reply org.tryton.Tryton-74.yml:4]: use org.flatpak.Builder from the docs linked in PR.

**Comment by bbhtt** [comment]
bot, build org.tryton.Tryton-74

**Review by hfiguiere — CHANGES_REQUESTED** [review]
each time we request that the icon, desktop and appstream file be managed upstream. And it is still not done?

**Comment by hfiguiere** [comment]
bot, build org.tryton.Tryton-74

**Comment by cedk** [comment]
> each time we request that the icon, desktop and appstream file be managed upstream. And it is still not done?

The Python packaging does not support to install them.

**Pull request closed without merging**